src/js/relay.firefox.com/get_profile_data.js

```diff
@@ -46,57 +46,36 @@
     const apiToken = profileMainElement.dataset.apiToken;
     await browser.storage.local.set({ apiToken });
 
+    const cookieString =
+    typeof document.cookie === "string" ? document.cookie : "";
+    
+    const cookieStringArray = cookieString
+      .split(";")
+      .map((individualCookieString) => individualCookieString.split("="))
+      .map(([cookieKey, cookieValue]) => [
+        cookieKey.trim(),
+        cookieValue.trim(),
+      ]);
+
+    const [_csrfCookieKey, csrfCookieValue] = cookieStringArray.find(
+      ([cookieKey, _cookieValue]) => cookieKey === "csrftoken"
+    );
+
+    await browser.storage.local.set({ csrfCookieValue: csrfCookieValue });
+
+
```

Comment:
I don't think you need a CSRF cookie if you're not making the request from the Relay website?

```suggestion
```

Reply:
I dug into the git-blame on the cookie string origins. It was me. 🪞👨 

https://github.com/mozilla/fx-private-relay-add-on/commit/200e2fbe6236fc5b2c32295cd7a62e6534e1750e

I think the initial reasoning was to verify if a user had the server-storage pref enabled? I'll remove this and confirm this doesn't break anything.

Reply:
See 25255d1bed74c5164629f9c406df84270edfb21d

Reply:
Ah OK, I wasn't aware that there was already code in `background.js` that read `csrfCookieValue` from the extension storage. AFAIK that's not needed, but it's outside of the scope of this PR then anyway.